### PR TITLE
fix(CSS): fix the document error

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:
1. fix the document error
2. fix lint suggest: var-naming 

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [] Tests added/passed.
* [X] Documentation updated.
* [] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccSomethingV0_basic'
...
=== RUN   TestAccSomethingV0_basic
--- PASS: TestAccSomethingV0_basic (70.75s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       70.796s
```

### CODE REVIEW

File: docs/resources/css_cluster.md [hunk 1]
```diff
@@ -110,10 +110,10 @@ The following arguments are supported:
   The [ess_node_config](#Css_ess_node_config) structure is documented below.
 
 * `master_node_config` - (Optional, List, ForceNew) Specifies the config of master node.
```
Comment (ShiChangkuo) [inline docs/resources/css_cluster.md]: all of the `xxx_node_config` blocks miss "Changing this ..."

File: docs/resources/css_cluster.md [hunk 2]
```diff
@@ -187,6 +187,33 @@ The `volume` block supports:
...
+* `flavor` - (Required, String, ForceNew) Specifies the flavor name. For example: value range of flavor ess.spec-2u8g:
+  40 GB to 800 GB, value range of flavor ess.spec-4u16g: 40 GB to 1600 GB, value range of flavor ess.spec-8u32g: 80 GB
+  to 3200 GB, value range of flavor ess.spec-16u64g: 100 GB to 6400 GB, value range of flavor ess.spec-32u128g: 100 GB
+  to 10240 GB. Changing this parameter will create a new resource.
+
+* `instance_number` - (Required, Int) Specifies the number of cluster instances.
+  + When it is `ess_node_config`, The value range is 1 to 200.
+  + When it is `master_node_config`, The value range is 3 to 10.
+  + When it is `client_node_config`, The value range is 1 to 32.
+  + When it is `cold_node_config`, The value range is 1 to 32.
```
Comment (ShiChangkuo) [inline docs/resources/css_cluster.md]:
```suggestion
  + When it is `master_node_config`, The value range is 3 to 10.
  + When it is `client_node_config`, The value range is 1 to 32.
```

File: docs/resources/css_cluster.md [hunk 3]
```diff
@@ -187,6 +187,33 @@ The `volume` block supports:
...
 * `size` - (Required, Int) Specifies the volume size in GB, which must be a multiple of 10.
 
+* `volume_type` - (Required, String, ForceNew) Specifies the volume type. COMMON: Common I/O. The SATA disk is used.
+  HIGH: High I/O. The SAS disk is used. ULTRAHIGH: Ultra-high I/O. The solid-state drive (SSD) is used. Changing this
+  parameter will create a new resource.
+
+<a name="Css_ess_node_config_forceNew"></a>
+The `master_node_config` and `client_node_config` block supports:
+
+* `flavor` - (Required, String, ForceNew) Specifies the flavor name. For example: value range of flavor ess.spec-2u8g:
```
Comment (ShiChangkuo) [inline docs/resources/css_cluster.md:195]: the example should be only for `master_node_config` and `client_node_config` block.

Comment (hmbig2) [reply docs/resources/css_cluster.md:195]: done

File: docs/resources/css_cluster.md (hunk 2 above)
Comment (hmbig2) [reply docs/resources/css_cluster.md]: done

File: docs/resources/css_cluster.md (hunk 1 above)
Comment (hmbig2) [reply docs/resources/css_cluster.md]: remove the forceNew from document and schema

**Pull request merged**